solutions.js

```diff
@@ -1,5 +1,39 @@
 console.log('Defining functions!');
+// 1.
 
 function changeNavColor() {
-  // change the navbar to a different color
+  var navBar = document.querySelector('#topbar');
+  navBar.style.backgroundColor = 'orange';
 };
+
+// 2.
+var belongToMe = document.createElement('h1');
+document.body.insertBefore(belongToMe, topbar);
```

Comment:
Watch your indents!

This should be in its own function (i.e. function `takeOver()`) and contain some text inside that `<h1>`.